qgreenland/util/qgis/metadata.py

```diff
@@ -26,7 +26,7 @@ def add_layer_metadata(map_layer: qgc.QgsMapLayer, layer_cfg: Layer) -> None:
     map_layer.setAbstract(tooltip)
 
     # Render the qmd template.
-    abstract = build_layer_abstract(layer_cfg)
+    abstract = escape(build_layer_abstract(layer_cfg))
```

Comment:
This function used to return xml escaped text. Since this function is also used to write the abstract info to the exported layer manifest file (`qgreenland.config.export.export_config_manifest`, a `json` file) and now the `metadata.txt` file, we should not escape for xml in all cases.

Reply:
:+1: